python/ql/src/Resources/FileNotAlwaysClosed.ql

```diff
@@ -52,7 +52,7 @@ predicate file_not_closed_at_scope_exit(ControlFlowNode open) {
 
 predicate file_not_closed_at_exception_exit(ControlFlowNode open, ControlFlowNode exit) {
     exists(EssaVariable v |
-        exit.(RaisingNode).viableExceptionalExit(_, _) and
+        exit.(RaisingNode).viableExceptionalExit_objectapi(_, _) and
```

Comment:
Old API?

Reply:
Not replacing things with the old API, but rather preserving the old API where the it's orthogonal to the query that the PR is for. `FileNotAlwaysClosed.ql` and others use predicates that share dependencies with `WrongNAmeForArgumentInCall.ql`, so either all the queries are updated in one fell swoop, or we keep some queries using the older predicates as we incrementally move things over.

For atomicity of PRs, I've done the latter. This is also what I've done in other queries. If these other queries had returned the right results, I would have probably just left them using the new API, but they don't, and it'll take a bunch of digging to figure out where the mismatch is coming from.

Reply:
Okay, this makes sense. I wasn't entirely clear that this was the PR that tried to do everything (and so that rolling things back here made sense). :+1: